capstone/src/test/java/com/google/sps/VoterServletTest.java

```diff
@@ -0,0 +1,41 @@
+package com.google.sps;
+
+import com.google.sps.servlets.VoterServlet;
+import java.io.IOException;
+import java.net.*;
+import java.util.*;
+import org.json.*;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class VoterServletTest {
+  @Test
+  public void rightOutput() throws IOException {
```

Comment:
rename: testGetJsonObjectSuccessfull()

And create a testGetEmptyJsonObject() for the other case